app/src/main/java/com/cmput301f20t21/bookfriends/ui/login/LoginActivity.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * LoginActivity.java
+ * Version: 1.0
+ * Date: October 18, 2020
+ * Copyright (c) 2020. Book Friends Team
+ * All rights reserved.
+ * github URL: https://github.com/CMPUT301F20T21/Book_Friends
+ */
+
+package com.cmput301f20t21.bookfriends.ui.login;
+
+import android.os.Bundle;
+import android.text.Editable;
+import android.view.View;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.cmput301f20t21.bookfriends.R;
+import com.google.android.material.textfield.TextInputEditText;
+import com.google.android.material.textfield.TextInputLayout;
+
+/**
+ * Activity that will display the login page and handle user inputs
+ */
+public class LoginActivity extends AppCompatActivity {
+    private TextInputLayout usernameLayout;
+    private TextInputLayout passwordLayout;
+    private TextInputEditText usernameField;
+    private TextInputEditText passwordField;
+
+    /**
+     * android lifecycle method, grab the Layout and EditText fields
+     * as well as adding a simple textChangedListener
+     * @param savedInstanceState - the saved objects, should contain nothing for this activity
+     */
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
```

Comment:
1. what if we [hide the action bar](https://stackoverflow.com/a/22338837/7358099) for login?
2. same for [the status bar](https://developer.android.com/training/system-ui/status#41)

Reply:
I think the app flows like this.
User open apps -> login activity -> main activity.
Action bar and status bar should only exist beyond main activity

Reply:
These codes under the `setContentView` should suffice to hide both status and action bar.
```
this.getSupportActionBar().hide();
View decorView = this.getWindow().getDecorView();
int uiOptions = View.SYSTEM_UI_FLAG_FULLSCREEN;
decorView.setSystemUiVisibility(uiOptions);
```

Reply:
the hiding status bar part in the above code is deprecated in API 30, but I cannot find alternatives for it. Using it as for now, both action bars and status bars are now hidden for login activity